sound/soc/codecs/rt715-sdca.h

```diff
@@ -0,0 +1,132 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/*
+ * rt715-sdca.h -- RT715 ALSA SoC audio driver header
+ *
+ * Copyright(c) 2019 Realtek Semiconductor Corp.
+*/
+
+#ifndef __RT715_SDCA_H__
+#define __RT715_SDCA_H__
+
+#include <linux/pm.h>
```

Comment:
why do you need pm.h? What you do need though is a forward declaration of `struct device` or `#include <linux/device.h>`